tests/pdf/test_pdf_export.py

```diff
@@ -39,6 +39,18 @@
             "/augsburg/ar/معلومات-الوصول/",
             "3b02f5ea5b/Integreat - Arabisch - معلومات الوصول.pdf",
         ),
+        (
+            "am",
+            [1, 2, 3, 4, 5, 6],
+            "/augsburg/am/እንኳን-ደህና-መጡ/",
+            "52d22a85dc/Integreat - Amharisch - እንኳን ደህና መጡ.pdf",
+        ),
+        (
+            "uk",
+            [1, 2, 3, 4, 5, 6],
```

Comment:
Here should be 4 pages, isn't it?

Reply:
Since the [`GeneratePdfView`](https://github.com/digitalfabrik/integreat-cms/blob/2e492ffa1f2de247019782342509429992ffc74a/integreat_cms/cms/views/pages/page_bulk_actions.py#L27) inherits from [`BulkActionView`](https://github.com/digitalfabrik/integreat-cms/blob/2e492ffa1f2de247019782342509429992ffc74a/integreat_cms/cms/views/bulk_action_views.py#L22), the selected ids are processed here:
https://github.com/digitalfabrik/integreat-cms/blob/2e492ffa1f2de247019782342509429992ffc74a/integreat_cms/cms/views/bulk_action_views.py#L87

Reply:
Ah, got it 👌